### ISSUE TITLE
Different Modifications at the Same Index

### ISSUE THREAD

**Issue by rfellers** [topdownproteomics/proteoform-nomenclature-standard#29]
Hey all,

1. I think it is fair to say that this ProForma term is valid:
`SEQVK[Acetyl|Unimod:1]ENCE`

2. So, given that, is the following ProForma term valid?
`SEQVK[Acetyl|Methyl]ENCE`

3. What about this example from the manuscript (where Label:13C(3) is a 3Da SILAC label)?
`SEQVK[Unimod:Label:13C(3)][Acetyl]ENCE`

I have my opinion, but I'll reserve that for the comments ...

Thanks,
Ryan

**Comment by acesnik** [comment]
Thanks for the question, Ryan. In Rule 2, we note that "Multiple modifications of the same amino acid are described by successive square bracket pairs." 
1. Therefore, I think the example in (1) is valid assuming the Unimod accession is 1. 
2. Example (2) doesn't satisfy rule 2; the pipe symbol separates mutliple descriptors of the same modification. 
3. Example (3) is valid.

**Comment by rfellers** [comment]
Interesting, I never really understood the difference between rule 2 and rule 4. That makes sense.

Sorry, I should have been more clear in my description as my real question was focused on something else. Correcting example 2 as `SEQVK[Acetyl][Methyl]ENCE`, is it valid for a proteoform to have 2 different modifications on the same spot? Is that chemically possible?

Example 3 is simply the same issue but with a label, which I believe is chemically possible (and common?).

**Comment by acesnik** [comment]
Yes, it is chemically possible to have up to three substituents on a primary amine, like on lysine in Example 2. Another more common example would be `SEQK[Methyl][Methyl]ENCE`. 

Example 3 is possible. An isotopic label doesn't change the chemistry of the lysine regarding modifications.

**Comment by rfellers** [comment]
Wow, I had no idea that was possible! I knew you could have 2 modifications on the N-terminus and that up to 3 methyl groups were possible on a single residue, but I didn't know the same was true for methyl and acetyl. Have you ever seen this (or something similar) in your data or in the literature?

**Comment by acesnik** [comment]
Yep! This is chemically possible, at least in an organic chemistry lab. (I discussed this with my colleagues here, to confirm.) We've never seen this in a biological sample, and I can't find it in such literature in a quick search. Enzymes that methylate or acetylate likely don't recognize already modified lysines.

**Comment by veitveit** [comment]
Agree, multiple PTMs need to be considered. These could be e.g. labelling
modifications on top of the usual PTMs.

Or PTMs that are differently labelled by for instance SILAC. By naming both
modifications separately, it is easier to avoid combinatorial explosions.

Does that make sense?
